core/store/src/db/rocksdb.rs

```diff
@@ -511,12 +482,15 @@ impl RocksDB {
         Checkpoint::new(&self.db).map_err(into_other)
     }
 
-    /// Gets every int property in CF_STAT_NAMES for every column in DBCol.
+    /// Gets every int property in CF_PROPERTY_NAMES for every column in DBCol.
     fn get_cf_statistics(&self, result: &mut StoreStatistics) {
-        for stat_name in CF_STAT_NAMES {
+        for prop_name in CF_PROPERTY_NAMES {
+            let stat_name = prop_name.to_bytes();
+            // SAFETY: RocksDB property names are always entirely ASCII.
+            let stat_name = unsafe { std::str::from_utf8_unchecked(stat_name) };
```

Comment:
I'd `std::str::from_utf8(stat_name).unwrap()` -- there's no reason to not have an assert here.

Reply:
Well, the reason is to avoid UTF8 check.  I think I’d rather turn CF_PROPERTY_NAMES into array of `(&str, &CStr)` if we really cannot stomach this unsafe.

Reply:
I think this unsafe is correct, but not warranted: this doesn't feel like a hot path, so there's not reason to skip what is essentially and assert, especially if that means typing unsafe. 

-0 mostly, this indeed is pretty tiny.